pkg/csi_driver/reconciler.go

```diff
@@ -1240,8 +1231,17 @@ func (recon *MultishareReconciler) listMultishareResourceOps(ctx context.Context
 			continue
 		}
 
+		klog.V(6).Infof("creation time: %s", meta.CreateTime)
 		var err error
 		if op.Error != nil {
+			// filter out error Op that's more than util.ErrRetention old
+			var createTime time.Time
+			createTime, err = time.Parse(util.OpTimeLayout, meta.CreateTime)
+			if err != nil {
+				klog.Errorf("failed to parse creation Time %q with error: %s", meta.CreateTime, err.Error())
+			} else if createTime.Before(time.Now().Add(-util.ErrRetention)) {
```

Comment:
1. why are we skipping ops before some threshold time?
2. how was 15 min decided?

Reply:
we are only skipping `if op.Error != nil`. 

For now it's arbitrary and doesn't do much. The thinking is that once the error is old, it's less relevant and we should reconcile the crds' `status.Error` field to prevent having error status on this shareInfo or instanceInfo forever and pushing up error counts.

But for now I haven't included the logic to actually remove the `status.Error`

Reply:
just to be safe we can also check (op.Done && op.Error != nil) in L1236